### ISSUE TITLE
The project is alive?

### ISSUE THREAD

**Issue by richardsonvix** [dot42/api#4]
I can create applications for Android?

**Comment by estebanjb** [comment]
yes, you can. the proyect is now open source. the problem is dot42 does not have official support any more
